packages/core/src/utilityApis.ts

```diff
@@ -88,3 +88,44 @@ export function matchTriggerQueries(
     return isTriggered(query, triggerQueries, options);
   };
 }
+
+// stub functions for better query parsing:
+
+type rootWordOptions = {
+  fuzzyMatch?: boolean, 
+  allowSynonyms?: boolean
+}
+
+export function hasAllRootWords(
+  query: string, 
+  words: Array<string>, 
+  options?: rootWordOptions
+): boolean {
+  return true;
+}
+
+export function hasAnyRootWords(
```

Comment:
For stubs, prefer to have specific input output pairs so that we know how the function is behaving.